### PR TITLE
Translate article 6, section 5

### PR DESCRIPTION
**Pull request by NahuelFerreroG**
# 🤘 Courses PRs

## Type

[x] Translation
[ ] Enhancement
[ ] New Article

### CODE REVIEW

File: dart/5.Command_line_app/6_pseudo_code_of_the_game.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# _Pseudo code_ for the game
+
+As we have been doing, let's write what we need to do in a colloquial way according to the requirements:
+
+```dart
+void main() {
+    print('Hello world!');
+
+
+    //  Pseudo code
```
Comment (maurodibert) [inline dart/5.Command_line_app/6_pseudo_code_of_the_game.md:10]: Is that possible and correct to align this like in the original file so we show different levels?

File: dart_sp/5.App_linea_comando/6_pseudo_code_del_juego.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # _Pseudo code_ para el juego
 
-Como venimos haciendo, escribamos lo que necesitamos hacer de forma coloquial en nuestro propio código  según los requerimientos:
+Como venimos haciendo, escribamos lo que necesitamos hacer de forma coloquial en nuestro propio código según los requerimientos:
```
Comment (maurodibert) [inline dart_sp/5.App_linea_comando/6_pseudo_code_del_juego.md:3]: Hawk eyes! 🦅

**Review by maurodibert — CHANGES_REQUESTED** [review]

File: dart/5.Command_line_app/6_pseudo_code_of_the_game.md (hunk 1 above)
Comment (NahuelFerreroG) [reply dart/5.Command_line_app/6_pseudo_code_of_the_game.md:10]: done

File: dart_sp/5.App_linea_comando/6_pseudo_code_del_juego.md (hunk 2 above)
Comment (NahuelFerreroG) [reply dart_sp/5.App_linea_comando/6_pseudo_code_del_juego.md:3]: jajaja

**Review by maurodibert — APPROVED** [review]

**Pull request merged**